internal/cmd/gocdk/launch.go

```diff
@@ -126,6 +127,16 @@ func newLauncher(ctx context.Context, pctx *processContext, launcherName string)
 			Client:       runService,
 			DockerClient: docker.New(pctx.env),
 		}, nil
+	case "ecs":
```

Comment:
These names (including the existing `cloudrun` are a bit obscure. It's fine if they are used as an internal enum, which I think these are, but we need more explanation (e.g., maybe links to product pages) when presenting to users.

Reply:
Ack. I agree, but I can't come up with an action to take on this feedback.

Reply:
the launcher interface could specify an "info" method, which would return a short paragraph, and it could be used in conjunction with the map of specifier strings to launcher types to display info to the user when the list of options is displayed?

Reply:
Great idea! Opened #2431 to track.